protocols/protocol/cloudstate/crdt.proto

```diff
@@ -328,43 +614,89 @@ message CrdtReply {
     bool streamed = 6;
 }
 
+//*CrdtStateAction* TODO
+//
+//---
+//
+//`Specification`:
+//
+//  - A `CrdtStateAction` is considered to be valid iff:
+//    - `action` MUST be one of the following:
+//      - `create` TODO
+//      - `update` TODO
+//      - `delete` TODO
+//    - `write_consistency` TODO
+//
+// FIXME set the ordinals to start at 1?
 message CrdtStateAction {
+
     oneof action {
+        // TODO
         CrdtState create = 5;
```

Comment:
It could be defined why the proxy does not expect a `CrdtState Create` message when a CRDT was initialised as a "new" CRDT and has no Delta after a command was handled.